express-post-json/index.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+
+const app = express();
+
+let nextId = 1;
+
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const gradesArray = [];
+  for (let id in grades) {
+    gradesArray.push(grades[id]);
+  }
+  res.json(gradesArray);
+})
+
+app.use(express.json());
```

Comment:
this needs to be above all of the endpoints

Reply:
![Screen Shot 2021-01-07 at 1 05 01 PM](https://user-images.githubusercontent.com/60412330/103944746-fcde6480-50e8-11eb-8317-4fdd22445836.png)